aspnetcore/tutorials/dotnet-watch.md

```diff
@@ -38,25 +38,27 @@ Application started. Press Ctrl+C to shut down.
 
 In a web browser, navigate to `http://localhost:<port number>/api/math/sum?a=4&b=5`. You should see the result of `9`.
 
-Navigate to the product API (`http://localhost:<port number>/api/math/product?a=4&b=5`). It returns `9`, not `20` as you'd expect. We'll fix that later in the tutorial.
+Navigate to the product API (`http://localhost:<port number>/api/math/product?a=4&b=5`). It returns `9`, not `20` as you'd expect. That problem is fixed later in the tutorial.
 
 ## Add `dotnet watch` to a project
```

Comment:
Could we use the same tactic as we did for the dotnet-user-secrets doc? i.e. `::: moniker range="<= aspnetcore-2.0"`

Reply:
Done. This section only appears for ASP.NET Core 2.0 and below.